app/misc/router.ts

```diff
@@ -311,6 +314,22 @@ function hideDiffPanelButtons() {
   disableDiffPanelEditOnHide();
 }
 
+function hideNavToolbar() {
+  // only hide these if there is no repo
+  if (!repoFullPath) {
+    document.getElementById("nav-repo-branch-tag-info")!.style.visibility = "hidden";
+    document.getElementById("nav-toolbar")!.style.visibility = "hidden";
+  }
+}
+
+function displayNavToolbar() {
+  // only show these if there is a repo
+  if (repoFullPath) {
+    document.getElementById("nav-repo-branch-tag-info")!.style.visibility = "visible";
+    document.getElementById("nav-toolbar")!.style.visibility = "visible";
```

Comment:
Maybe you should try using the display property?

`document.getElementById("nav-repo-branch-tag-info")!.style.display = "none";` to hide
`document.getElementById("nav-repo-branch-tag-info")!.style.display = "block";` to show

Reply:
I don't think it's an issue with the method. I think it's an issue with the repoFullPath?